index.html

```diff
@@ -46,4 +46,238 @@ <h1 class="mola">  I'm Sayed Ramin.<br />Glad to see<br />
+          alt="background"
+        />
+      </a>
+      <div class="pra">
+        <h3 class="title1">Tonic</h3>
+        <ul class="flex">
+          <li class="canopy-">CANOPY</li>
+          <li class="p"> <img src="image/works/Counter.png"></li>
+          <li class="role">Back End Dev</li>
+          <li class="p"> <img src="image/works/Counter.png"></li>
+          <li class="year">2015</li>
+        </ul>
+        <p class="text1">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="btn1">
+          <li class="btn">
+            <button class="html" type="button">HTML</button>
+          </li>
+          <li class="btn">
+            <button class="css" type="button">CSS</button>
+          </li>
+          <li class="btn">
+            <button class="java" type="button">JAVASCRIPT</button>
+          </li>
+        </ul>
+        <div class="btn2">
+          <button type="button" class="btn-see">See Project</button>
+        </div>
+      </div>
+    </div>
+    <section class="page2">
+      <div class="@continer">
+    
+        <div class="layout1">
+          <a href="#">
+            <img
+              class="imagework"
+              src="image/works/Snapshoot Portfolio (3).png"
+              alt="background"
+            />
+          </a>
+          <div class="pra">
+            <h3 class="title1">Multi-Post</br> Stories </h3></br></h3>
+            <ul class="flex">
+              <li class="canopy-">CANOPY</li>
+              <li class="p"> <img src="image/works/Counter.png"></li>
+              <li class="role">Back End Dev</li>
+              <li class="p"> <img src="image/works/Counter.png"></li>
+              <li class="year">2015</li>
+            </ul>
+            <p class="text1">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="btn1">
+              <li class="btn">
+                <button class="html" type="button">HTML</button>
+              </li>
+              <li class="btn">
+                <button class="css" type="button">CSS</button>
+              </li>
+              <li class="btn">
+                <button class="java" type="button">JAVASCRIPT</button>
+              </li>
+            </ul>
+            <div class="btn2">
+              <button type="button" class="btn-see">See Project</button>
+            </div>
+          </div>
+        </div>
+ <!--Works Section page 4-->
+ <section class="page4">
+  <div class="@continer">
+      </a>
+      <div class="pra">
+        <h3 class="titleA">About My self</h3>
+        <p class="text3">
+          Hello I’m a software developer! I can<br> help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
```

Comment:
Kindly update your about myself section to match the figma design better🙏

Your design:
<img width="143" alt="image" src="https://github.com/sadat-Ramin/My_first_portfolio/assets/37116322/86fa6317-8b4c-40b8-bf86-53ac6e1e1d65">

Figma design:
<img width="156" alt="image" src="https://github.com/sadat-Ramin/My_first_portfolio/assets/37116322/6c2c3509-600f-4154-a3b2-962c1540c5de">

A few things to note:
- Please add the border raius at he top right of the section and make sure its easy to see where the worst section  ends and the about section starts
- Kindly increase the spacing between the technology categories and the arrow icons 
- Please increase the padding inside each of the technologies (HTML, CSS, JS)